index.html

```diff
@@ -1,12 +1,157 @@
+        </ul>
+      </div>
+
+      <div>
+        <p>
+          Интернет-магазин <br />
+          строительных материалов <br />
+          и инструментов для ремонта
+        </p>
+
+        <a href="tel:+78125550555">7(812)555-05-55 </a>
+        <p>г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
+        <a href="">Войти</a>
+        <a href="">Регистрация</a>
+      </div>
+
+      <nav>
+        <ul>
+          <li><a href="">Главная</a></li>
+          <li><a href="">Компания</a></li>
+          <li><a href="catalog.html">Каталог</a></li>
+          <li><a href="">Новости</a></li>
+          <li><a href="">Спецпредложения</a></li>
+          <li><a href="">Доставка</a></li>
+          <li><a href="">Контакты</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h2>Компания «Техномарт»</h2>
+
+        <p>
+          Настоящий мужской шоппинг начинается здесь! Огромный выбор товаров для
+          <br />
+          ремонта и строительства не оставит равнодушным любителей сэкономить на
+          <br />
+          гастарбайтерах.
+        </p>
+
+        <p>Мы можем доставить ваш товар в самые отдаленные точки России!</p>
+        <p>Техномарт работает со многими транспортными компаниями:</p>
+
+        <ul>
+          <li>Деловые линии</li>
+          <li>Автотрейдинг</li>
+          <li>ЖелДорЭкспедиция</li>
+        </ul>
+
+        <p><a href=""> Подробнее о компании </a></p>
+      </section>
+
+      <section>
+        <h2>Контакты</h2>
+        <p>
+          Вы можете забрать товар сами,<br />
+          заехав в наш офис:
+        </p>
+
+        <img src="https://via.placeholder.com/150x100" alt="" />
+
+        <button>Заблудились? Напишите нам!</button>
+      </section>
+    </main>
+
+    <footer>
+      <div>
+        <div>
+          <a href="/index.html">
+            <img
+              src="/img/логотип.svg"
+              width="220"
+              height="47"
+              alt="Логотип магазина Техномарт"
+            />
+          </a>
+          <p>г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
+          <a href="tel:+78125550555">+7 (812) 555-05-55</a>
+        </div>
+
+        <div>
+          <ul>
+            <li><a href="">Компания</a></li>
+            <li><a href="">Новости</a></li>
+            <li><a href="catalog.html">Каталог</a></li>
+            <li><a href="">Доставка</a></li>
+            <li><a href="">Контакты</a></li>
+          </ul>
+
+          <ul>
+            <li><a href="">Материалы</a></li>
+            <li><a href="">Техника</a></li>
+            <li><a href="">Инструмент</a></li>
+            <li><a href="">Спецпредложения</a></li>
+          </ul>
+        </div>
+      </div>
+      <div>
+        <div>
+          <p>
+            © 2010-2019 Компания «Техномарт» <br />
+            Все права защищены
+          </p>
+        </div>
+
+        <div>
+          <ul>
+            <li><a href=""></a>Vkontakte</li>
```

Comment:
```suggestion
            <li><a href="">Vkontakte</a></li>
```
Посмотри в статье: https://htmlacademy.ru/blog/boost/graphics/content-or-decor-img блок Мы в соц сетях, там рассказано как делать соц сеточки

Reply:
прочитал статью, там написано, что надо применять background-image в ссылке, но насколько я понял, это же в css стилях делается, так что не особо понял, что от меня тут требуется. Ошибку с положением текста понял, перенес внутрь ссылки

Reply:
> Таким образом, здесь применяется комплексный подход: **внутри ссылки обязательно прописывается поясняющий текст**.

Reply:
текст внутри. Что ты и сделал.